torch/_refs/__init__.py

```diff
@@ -949,6 +950,45 @@ def trunc(a):
     return prims.trunc(a)
 
 
+# TODO: register this as a real ref/decomposition once TorchInductor supports complex!
+def view_as_complex(self: TensorLikeType) -> TensorLikeType:
+    input_dtype = self.dtype
+    if utils.is_complex_dtype(input_dtype):
+        return self
```

Comment:
@Fidget-Spinner why did you add this? It looks to me like `view_as_complex` of a complex tensor is not legal. Was there a test failing?

Reply:
In eager I get:
```
RuntimeError: view_as_complex is only supported for half, float and double tensors, but got a tensor of scalar type: ComplexFloat
```

Reply:
Woops. That was a remnant from a direct translation of ``c10::toComplexType``. Sorry removed!